src/Plugins/SimplnxCore/test/AppendImageGeometryTest.cpp

```diff
@@ -29,6 +29,9 @@ const DataPath k_CroppedBottomYPath({"CroppedBottomY"});
 const DataPath k_CroppedMiddleYPath({"CroppedMiddleY"});
 const DataPath k_CroppedTopYPath({"CroppedTopY"});
 const DataPath k_AppendedGeometryPath({"AppendedGeometry"});
+inline constexpr StringLiteral k_MirroredXExemplarDCName("MirroredXDataContainer");
+inline constexpr StringLiteral k_MirroredYExemplarDCName("MirroredYDataContainer");
+inline constexpr StringLiteral k_MirroredZExemplarDCName("MirroredZDataContainer");
```

Comment:
```suggestion
constexpr StringLiteral k_MirroredXExemplarDCName("MirroredXDataContainer");
constexpr StringLiteral k_MirroredYExemplarDCName("MirroredYDataContainer");
constexpr StringLiteral k_MirroredZExemplarDCName("MirroredZDataContainer");
```

No `inline` keyword in any `.cpp` files